app/components/Snaps/SnapUILink/SnapUILink.tsx

```diff
@@ -29,21 +29,22 @@ const onPress = (href: string) => {
 // TODO: This component should show a modal for links when not using preinstalled Snaps
 export const SnapUILink: React.FC<SnapUILinkProps> = ({ href, children }) => {
   const label = (
-    <>
+    <View
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: 2,
```

Comment:
```suggestion
        gap: 4,
```
To match the extension design